etc/spack/defaults/config.yaml

```diff
@@ -164,7 +164,7 @@ config:
   #
   # 'clingo' currently requires the clingo ASP solver to be installed and
   # built with python bindings. 'original' is built in.
-  concretizer: original
+  concretizer: clingo
```

Comment:
We should update the comment here.

Reply:
Comment updated in 549f446 Let me know if that's good or should be rephrased.